url_screenshot/icon_url_screenshot/actions/screenshot/action.py

```diff
@@ -0,0 +1,52 @@
+import komand
+from .schema import ScreenshotInput, ScreenshotOutput, Input, Output, Component
+# Custom imports below
+from komand.exceptions import PluginException
+import os
+import uuid
+import base64
+import time
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+
+class Screenshot(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='screenshot',
+                description=Component.DESCRIPTION,
+                input=ScreenshotInput(),
+                output=ScreenshotOutput())
+
+    def run(self, params={}):
+        CHROME_PATH = '/usr/bin/chromium'
+        CHROMEDRIVER_PATH = '/usr/bin/chromedriver'
+        WINDOW_SIZE = "1920,1080"
+
+        name = "/tmp/{}{}".format(str(uuid.uuid1()), ".png")
```

Comment:
Take a look at f strings https://realpython.com/python-f-strings/

Reply:
yeah, f strings are shorter and potentially cleaner, and since we control the entire python environment via docker...... no reason not to use them!  Good catch

Reply:
reworked - old habits die hard